yarl/_url.py

```diff
@@ -701,11 +683,45 @@ def _validate_authority_uri_abs_path(host, path):
                 "Path in a URL with authority should start with a slash ('/') if set"
             )
 
+    def _make_child(self, segments, encoded=False):
+        # add segments to self._val.path, accounting for absolute vs relative paths
```

Comment:
Why isn't this a docstring? It explains what the code does. Comments are supposed to explain motivation/unobvious decisions but the behavior description should go into a docstring.

Reply:
Not sure, I think I meant it to be a docstring..